spec.bs

```diff
@@ -607,6 +608,10 @@ members:
               language or direction metadata associated with specific input
               members, in that it represents the caller's requested localized
               experience rather than assertion about a specific string value.
+
+    :  <dfn>showOptOut</dfn> member
+    :: Whether the user should be shown a button to opt-out. Optional, default
```

Comment:
We should not presume a specific UX (e.g., a button), and instead try to write for the general case.

This can be quite tricky, but you can see an example in https://w3c.github.io/secure-payment-confirmation/#sctn-transaction-confirmation-ux where we say today "the User Agent MUST ensure that the following is communicated to the user and that the user’s consent is collected for the authentication", rather than say (for example) "the User Agent MUST display a dialog with the following text fields".

For opt-out, I think the wording might be something like:

`"the user should be given a choice to opt-out during the [Transaction UX]"`

This comment applies to all references to 'button' below :)

Reply:
Done